src/libs/actions/Report.js

```diff
@@ -1183,7 +1183,7 @@ function updateLastReadActionID(reportID, sequenceNumber) {
 
     // Need to subtract 1 from sequenceNumber so that the "New" marker appears in the right spot (the last read
     // action). If 1 isn't subtracted then the "New" marker appears one row below the action (the first unread action)
-    const lastReadSequenceNumber = (sequenceNumber - 1) || reportMaxSequenceNumbers[reportID];
+    const lastReadSequenceNumber = sequenceNumber !== null ? (sequenceNumber - 1) : reportMaxSequenceNumbers[reportID];
```

Comment:
Should this be !_.isNull? Regardless, please update the comment to explain this

Reply:
Made a better improvement. Additionally, there isn't much value in the comment since if sequenceNumber is provided we want to use it else not use it. But either way i added something to explain that its okay for lastReadSequenceNumber to be 0.